docs/book/src/tasks/bootstrap/kubeadm-bootstrap/index.md

```diff
@@ -41,7 +41,6 @@ apiVersion: bootstrap.cluster.x-k8s.io/v1beta2
 metadata:
   name: my-control-plane1-config
 spec:
```

Comment:
We also have to drop spec in these cases. This is not valid YAML, additionally an empty spec is not allowed (no spec is allowed)